vcxagency-node/src/scripts/download-certs.js

```diff
@@ -0,0 +1,44 @@
+const { S3Client, GetObjectCommand } = require('@aws-sdk/client-s3')
+const fs = require('fs')
+const logger = require('../logging/logger-builder')(__filename)
+
+module.exports.fetchAwsAssets = async function fetchAwsAssets (appConfig) {
```

Comment:
I'd call it `fetchCertsFromS3`, `fetchAwsAssets ` is quite general name, but the implementation does something very specific.

Reply:
Agree